staging/src/k8s.io/apiserver/pkg/authentication/token/tokenfile/tokenfile_test.go

```diff
@@ -107,6 +138,26 @@ token7,user7,uid7,"group1,group2",otherdata
 	}
 }
 
+func TestBadTokenFileMustHaveTenant(t *testing.T) {
```

Comment:
the test case name is a bit weird to me. I guess you mean BadTokenFileMIssingTenant?

Reply:
good call. I'll use BadTokenFileMIssingRequiredTenant